client/config/wdio.config.js

```diff
@@ -110,7 +110,7 @@ const config = {
   waitforTimeout: 10000,
   //
   // Default timeout in milliseconds for request
-  // if Selenium Grid doesn't send response
+  // if Selenium Container doesn't send response
```

Comment:
I think your change of 'Grid' to 'Container' was a bit too agressive (see also e.g. `client/src/models/Location.js` and `client/tests/models/geoLocation.test.js`). :wink: